build.gradle.kts

```diff
@@ -174,25 +222,30 @@ tasks.register("createMsi") {
       processBuilder1.command(parameters)
       val process1 = processBuilder1.start()
       if (process1.waitFor() != 0) {
-        throw GradleException("Error while creating the .msi package")
+        throw GradleException("Error while creating the MSI package file.")
```

Comment:
`MSI package` is a kind of file, so the "file" at the end of the line is redundant.